### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by ivan-nginx**
Enjoy!

### CODE REVIEW

File: languages/zh-CN.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ post:
   untitled: 未命名
   toc_empty: 此文章未包含目录
   views: 阅读次数
-  comments_count: Comments
+  comments_count: 评论
```
Comment (sli1989) [inline languages/zh-CN.yml]: how about `评论数`

Comment (tsanie) [reply languages/zh-CN.yml]: maybe good, changed.

**Comment by ivan-nginx** [comment]
@wafer-li @maple3142 cn-HK, cn-TW?

File: languages/zh-CN.yml (hunk 1 above)
Comment (geekrainy) [reply languages/zh-CN.yml]: Is this change more appropriate?

File: languages/zh-HK.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ post:
   untitled: 未命名
   toc_empty: 此文章未包含目錄
   views: 閱讀次數
-  comments_count: 評論數
+  comments_count: 評論
```
Comment (geekrainy) [inline languages/zh-HK.yml]: Is this change more appropriate?

File: languages/zh-TW.yml [hunk 3]
```diff
@@ -29,7 +29,7 @@ post:
   untitled: 未命名
   toc_empty: 此文章沒有目錄
   views: 閱讀次數
-  comments_count: 評論數
+  comments_count: 評論
```
Comment (geekrainy) [inline languages/zh-TW.yml]: Is this change more appropriate?

**Comment by ivan-nginx** [comment]
@geekrainy can u register in https://crowdin.com?

**Comment by geekrainy** [comment]
@ivan-nginx I just registered it. Nickname: geekrainy

**Comment by ivan-nginx** [comment]
@geekrainy invited. U can change languages there. No need to make any changes to `i18n` branch, it's just autmatic branch by this service.

File: languages/zh-HK.yml (hunk 2 above)
Comment (sli1989) [reply languages/zh-HK.yml]: In `zh-CN.yml`, we used `评论数`. So need a decision before v6.1.0 realease.

**Pull request merged**